Extensions/audio_plus.css

```diff
@@ -33,36 +33,103 @@
 	border-radius: 6px;
 }
 
-.xkit-audio-plus-controls {
+#right_column.has_docked_audio {
+	opacity: 0.3;
+}
+
+.xkit-audio-plus-pseudo-post {
+	display: block;
 	position: fixed;
-	bottom: 50px;
-	left: 50%;
-	transform: translateX(245px) translateY(0px);
+	background: white;
+	height: auto;
+	width: 300px;
+	border-radius: 3px;
+	padding: 15px 0;
+	left: calc(50% + 178px);
+	bottom: 15px;
+	opacity: 0;
+	z-index: -1;
+	transition:
+		bottom .15s cubic-bezier(.165,.84,.44,1),
+		transform .15s cubic-bezier(.165,.84,.44,1), 
+		opacity .15s cubic-bezier(.165,.84,.44,1);
+}
 
-	z-index: 10000;
-	width: 85px;
+#search_actions_search .xkit-audio-plus-pseudo-post {
```

Comment:
the id for the page u get when you search smth from the header

when you set up a list view (instead of the brick view) you get smth that looks like the dashboard, but is slightly to the right, so this rule just keeps the spacing consistent ^__^